dataprofiler/tests/profilers/test_numeric_stats_mixin_profile.py

```diff
@@ -528,4 +528,4 @@ def test_diff_helper(self):
             'variance': -8.362573099415204,
             'stddev': -2.0238425028660023
         }
-        self.assertDictEqual(expected_diff, other1._diff_helper(other2))
+        self.assertDictEqual(expected_diff, other1.diff(other2))
```

Comment:
You should rename this test function `test_diff` now since its no longer called `_diff_helper`

Reply:
agree with the recommended change